students/piotrowski_stanislaw/lesson_03_functions/calculator.py

```diff
@@ -0,0 +1,88 @@
+# Calculator.py
+# piotrsta
+
+
+def help_calc():
+    print("a - add")
+    print("s - subtract")
+    print("m - multiply")
+    print("d - divide")
+    print("p - power")
+    print("h,? - help")
+    print("q - QUIT")
+
+
+def add(x, y):
+    print("Result: " + str(x + y))
+
+
+def subtract(x, y):
+    print("Result: " + str(x - y))
+
+
+def multiply(x, y):
+    print("Result: " + str(x * y))
+
+
+def divide(x, y):
+    print("Result: " + str(x / y))
+
+
+def power(x, y):
+    print("Result: " + str(x ** y))
+
+
+def input_operand1():
+    operand1 = int(input("Input 1st operand: "))
+    return operand1
+
+
+def input_operand2():
+    operand2 = int(input("Input 2nd operand: "))
+    return operand2
+
+
+print("Welcome to NEW calculator:")
+help_calc()
+
+while True:
+    print("Enter option:")
+    option = input()
+
+    if option == "a":
+        print("ADDING")
+        x = input_operand1()
+        y = input_operand2()
+        add(x, y)
+
+    if option == "s":
```

Comment:
jako że w tym przypadku mamy wybór jednej z opcji, to lepiej jest użyc konstrukcji `if elif`, jest to odpowiednik instrukcji `case` w innych językach.
Przy aktualnym rozwiązaniu, gdy użytkownik wybierze 'a' to program sprawdzi każdy warunek do końca zamiast wrócić na początek.
Nie zapomnij również coś zakomunikować użytkownikowi, gdy wybierze nie obsługiwaną opcję.